build/index.sh

```diff
@@ -78,6 +78,14 @@ unzip -o -d ../bin -j darwin-x64.zip ffmpeg
 mv ../bin/ffmpeg ../bin/darwin-x64
 curl -s -L 'https://evermeet.cx/ffmpeg/info/ffmpeg/release' | jq --tab '.' >../bin/darwin-x64.README
 
+echo 'darwin arm64'
+echo '  downloading from osxexperts.net'
+download 'https://www.osxexperts.net/ffmpeg44arm.zip' darwin-arm64.zip
+echo '  extracting'
+unzip -o -d ../bin -j darwin-arm64.zip ffmpeg
+mv ../bin/ffmpeg ../bin/darwin-arm64
+curl -s -L 'http://osxexperts.net/'  -o ../bin/darwin-arm64.README
```

Comment:
We should find a better text to serve as `darwin-arm64.README`. Maybe just put a small file with a link to this website?

Also, strictly speaking, the ffmpeg license(s) make it mandatory to distribute a license file with the binary.

Reply:
I'm now pulling the README and LICENSE straight from the ffmpeg source, is that alright?

on a side note should I also add a license to the darwin x64 build commands?

Reply:
> I'm now pulling the README and LICENSE straight from the ffmpeg source, is that alright?

Yeah, I'd say that's sufficient.

> on a side note should I also add a license to the darwin x64 build commands?

Yes, this will make them show up when people install `ffmpeg-static`.

Reply:
sounds good, added LICENSE to `darwin-x64`